x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/analysis/AnalyzerTests.java

```diff
@@ -2190,6 +2191,36 @@ public void testLookupJoinUnknownField() {
         assertThat(e.getMessage(), containsString(errorMessage3 + "right side of join"));
     }
 
+    public void testMultipleLookupJoinsGiveDifferentAttributes() {
```

Comment:
@craigtaverner , we were chatting about attributes obtained from different `LOOKUP JOINs`. This test demonstrates that they do the right thing!